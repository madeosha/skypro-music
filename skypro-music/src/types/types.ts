export type SignInFormType = {
  email: string;
  password: string;
};

export type SignUpFormType = {
  email: string;
  password: string;
  username: string;
};

export type UserType = {
  id: number;
  username: string;
  firstName: string;
  lastName: string;
  email: string;
};
