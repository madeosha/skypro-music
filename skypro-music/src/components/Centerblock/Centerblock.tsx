import { Filter } from "../Filter/Filter";
import { Search } from "../Search/Search";
import styles from "./Centerblock.module.css";
import classNames from 'classnames';

export const Centerblock = () => {
    return (
    <div className={styles.main_centerblock}>
            <Search />
            <h2 className={styles.centerblock__h2}>Треки</h2>
            <Filter />
            <div className={styles.centerblock__content}>
              <div className={styles.content__title}>
                <div className={classNames(styles.playlist_title__col, styles.col01)}>Трек</div>
                <div className={classNames(styles.playlist_title__col, styles.col02)}>Исполнитель</div>
                <div className={classNames(styles.playlist_title__col, styles.col03)}>Альбом</div>
                <div className={classNames(styles.playlist_title__col, styles.col04)}>
                  <svg className={styles.playlist_title__svg}>
                    <use href="img/icon/sprite.svg#icon-watch"></use>
                  </svg>
                </div>
              </div>
              <div className={styles.content__playlist}>
                
                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Guilt 
                            <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Nero</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Welcome Reality</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>4:44</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Elektro 
                            <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Dynoro, Outwork, Mr. Gee</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Elektro</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__title_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>2:22</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">I’m Fire 
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Ali Bakgor</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">I’m Fire</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>2:22</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Non Stop
                          <span className={styles.track__title_span}>(Remix)</span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Стоункат, Psychopath</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Non Stop</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>4:12</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Run Run
                          <span className={styles.track__title_span}>(feat. AR/CO)</span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Jaded, Will Clarke, AR/CO</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Run Run</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>2:54</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Eyes on Fire
                          <span className={styles.track__title_span}>(Zeds Dead Remix)</span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Blue Foundation, Zeds Dead</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Eyes on Fire</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>5:20</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Mucho Bien
                          <span className={styles.track__title_span}>(Hi Profile Remix)</span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">HYBIT, Mr. Black, Offer Nissim, Hi Profile</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Mucho Bien</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>3:41</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Knives n Cherries
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">minthaze</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Captivating</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>1:48</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">How Deep Is Your Love
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Calvin Harris, Disciples</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">How Deep Is Your Love</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>3:32</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">Morena
                          <span className={styles.track__title_span}></span>
                        </a>
                      </div>
                   </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://">Tom Boxer</a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://">Soundz Made in Romania</a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}>3:36</span>
                    </div>
                  </div>
                </div>

                <div className={styles.playlist__item}>
                  <div className={styles.playlist__track}>
                    <div className={styles.track__title}>
                      <div className={styles.track__title_image}>
                        <svg className={styles.track__title_svg}>
                          <use href="img/icon/sprite.svg#icon-note"></use>
                        </svg>
                      </div>
                      <div className={styles.track__title_text}>
                        <a className={styles.track__title_link} href="http://">
                          <span className={styles.track__title_span}></span>
                        </a>                                                                                                                                                                                                                                                                                                                          
                      </div>
                    </div>
                    <div className={styles.track__author}>
                      <a className={styles.track__author_link} href="http://"></a>
                    </div>
                    <div className={styles.track__album}>
                      <a className={styles.track__album_link} href="http://"></a>
                    </div>
                    <div className={styles.track__time}>
                      <svg className={styles.track__time_svg}>
                        <use href="img/icon/sprite.svg#icon-like"></use>
                      </svg>
                      <span className={styles.track__time_text}></span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
);
}

