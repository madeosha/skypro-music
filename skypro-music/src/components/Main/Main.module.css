.wrapper {
    width: 100%;
    min-height: 100%;
    overflow: hidden;
    background-color: #383838;
}

.container {
    max-width: 1920px;
    height: 100vh;
    margin: 0 auto;
    position: relative;
    background-color: #181818;
}

.main {
    -webkit-box-flex: 1;
    -ms-flex: 1 1 auto;
    flex: 1 1 auto;
    display: -webkit-box;
    display: -ms-flexbox;
    display: flex;
    -ms-flex-wrap: wrap;
    flex-wrap: wrap;
    -webkit-box-pack: justify;
    -ms-flex-pack: justify;
    justify-content: space-between;
}

._btn_text:hover {
    border-color: #d9b6ff;
    color: #d9b6ff;
    cursor: pointer;
}

._btn_icon:hover svg {
    fill: transparent;
    stroke: #acacac;
    cursor: pointer;
}

._btn_text:active {
    border-color: #ad61ff;
    color: #ad61ff;
    cursor: pointer;
}

._btn_icon:active svg {
    fill: transparent;
    stroke: #ffffff;
    cursor: pointer;
}

._btn_icon:active .track_play__like_svg,
._btn_icon:active .track_play__dislike_svg {
    fill: #696969;
    stroke: #ffffff;
    cursor: pointer;
}