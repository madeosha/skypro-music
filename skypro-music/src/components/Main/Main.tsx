import { Centerblock } from "../Centerblock/Centerblock";
import { Menu } from "../Menu/Menu";
import { Player } from "../Player/Player";
import { Sidebar } from "../Sidebar/Sidebar";
import styles from "./Main.module.css";

export const Main = () => {
    return (
        <div className={styles.wrapper}>
        <div className={styles.container}>
            <main className={styles.main}>
                <Menu />
                <Centerblock />
                <Sidebar />
            </main>
            <Player />
            <footer className={styles.footer}></footer>
        </div>
      </div>
    )
}