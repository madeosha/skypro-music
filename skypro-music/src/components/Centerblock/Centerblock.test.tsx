import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";
import Centerblock from "./Centerblock";
import { Track } from "../Main/Main.types";

// Создаем mock store
const mockStore = configureStore([]);

describe("Корректный рендер компонента Centerblock", () => {
  let store: any;

  beforeEach(() => {
    store = mockStore({
      player: {
        currentTrack: null,
        isPlaying: false,
      },
    });
  });

  const allTracks: Track[] = [
    {
      id: 1,
      name: "Track 1",
      author: "Author 1",
      release_date: "2023-01-01",
      genre: "Pop",
      duration_in_seconds: 180,
      album: "Album 1",
      logo: "path/to/logo1.jpg",
      track_file: "path/to/track1.mp3",
      stared_user: ["user1"],
    },
    {
      id: 2,
      name: "Track 2",
      author: "Author 2",
      release_date: "2023-02-01",
      genre: "Rock",
      duration_in_seconds: 200,
      album: "Album 2",
      logo: "path/to/logo2.jpg",
      track_file: "path/to/track2.mp3",
      stared_user: ["user2", "user3"],
    },
  ];

  it("Корректный рендер компонента Centerblock с треками", () => {
    const component = renderer.create(
      <Provider store={store}>
        <Centerblock allTracks={allTracks} error={null} isLoading={false} />
      </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Дополнительный тест для проверки рендера состояния загрузки", () => {
    const component = renderer.create(
      <Provider store={store}>
        <Centerblock allTracks={[]} error={null} isLoading={true} />
      </Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
