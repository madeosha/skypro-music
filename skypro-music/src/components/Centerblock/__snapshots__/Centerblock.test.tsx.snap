// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Корректный рендер компонента Centerblock Дополнительный тест для проверки рендера состояния загрузки 1`] = `
<div
  className="centerblock__content"
>
  <div
    className="content__title"
  >
    <div
      className="playlist_title__col col01"
    >
      Трек
    </div>
    <div
      className="playlist_title__col col02"
    >
      Исполнитель
    </div>
    <div
      className="playlist_title__col col03"
    >
      Альбом
    </div>
    <div
      className="playlist_title__col col04"
    >
      <svg
        className="playlist_title__svg"
      >
        <use
          href="/img/icon/sprite.svg#icon-watch"
        />
      </svg>
    </div>
  </div>
  <div
    className="content__playlist"
  >
    <span>
      Треки не найдены
    </span>
  </div>
</div>
`;

exports[`Корректный рендер компонента Centerblock Корректный рендер компонента Centerblock с треками 1`] = `
<div
  className="centerblock__content"
>
  <div
    className="content__title"
  >
    <div
      className="playlist_title__col col01"
    >
      Трек
    </div>
    <div
      className="playlist_title__col col02"
    >
      Исполнитель
    </div>
    <div
      className="playlist_title__col col03"
    >
      Альбом
    </div>
    <div
      className="playlist_title__col col04"
    >
      <svg
        className="playlist_title__svg"
      >
        <use
          href="/img/icon/sprite.svg#icon-watch"
        />
      </svg>
    </div>
  </div>
  <div
    className="content__playlist"
  >
    <span>
      Треки не найдены
    </span>
  </div>
</div>
`;
